client/components/data/query-post-stats/index.jsx

```diff
@@ -8,60 +8,39 @@ import { isRequestingPostStats } from 'calypso/state/stats/posts/selectors';
 class QueryPostStats extends Component {
 	static defaultProps = {
 		requestPostStats: () => {},
```

Comment:
`requestPostStats` is a connected prop that's always present, the default is redundant.